src/app/shared/components/template/components/layout/display-group/sticky/display-group-sticky.component.scss

```diff
@@ -1,10 +1,8 @@
 // When the display group is "sticky", style it as an inline header/footer
 :host {
-  display: flex;
```

Comment:
question: have you checked whether offsets still calculate correctly with this property removed. I expect we might need the default display to be set as `block` for the `#sticky.contentHeight()` property to evaluate correctly